lms/djangoapps/django_comment_client/utils.py

```diff
@@ -677,11 +677,19 @@ def permalink(content):
     else:
         course_id = content['course_id']
     if content['type'] == 'thread':
-        return reverse('discussion.views.single_thread',
-                       args=[course_id, content['commentable_id'], content['id']])
+        try:
+            return reverse('discussion.views.single_thread',
```

Comment:
Locally `reverse('discussion.views.single_thread'` was throwing the exception but `reverse('single_thread',` worked fine.
We don't use this feature actively but still, just to be on safe side

Reply:
I am open to suggestions, if something better could be done

Reply:
What if we just replace 'discussion.views.single_thread'  with 'single_thread' ?

Reply:
Don't want to break anything if it works on cloud as it is 😓